force_wfmanager/left_side_pane/tree_pane.py

```diff
@@ -67,12 +68,17 @@ class TreePane(TraitsDockPane):
     #: Enable or disable the run button.
     run_enabled = Bool(True)
 
+    plugin_data_views = List
+
+    selected_data_view = Enum(values='plugin_data_views')
```

Comment:
It would be nice if this UI showed a human-readable label/description for the DataView rather than its path.

Reply:
Done